openQA.spec

```diff
@@ -527,6 +529,7 @@ fi
 %{_datadir}/openqa/lib/OpenQA/Client
 %{_datadir}/openqa/lib/OpenQA/UserAgent.pm
 %{_bindir}/openqa-client
+%{_bindir}/openqa-cli
```

Comment:
With the two names so similar auto-completion will be tricky as "openqa-cli" is now ambiguous. I don't have a good proposal how to prevent this. Maybe you have an idea?

Reply:
You have already mentioned one proposal: "I am now even more inclined towards replacing the existing client script content rather than adding another script."

I wouldn't care too much about it. At least Bash will stop the completing "at" `openqa-cli`. That means the new client will be thrown at users though the shell completion but I don't think that's a bad thing. Not sure what's tricky about it. Users can just use `tab+tab` to see the options.

Reply:
One way to go about it would be to call it `openqa-client-ng` and at some point in the future replace `openqa-client.

Reply:
Isn't the existing client installed simply as `client`?

Reply:
No, it's /usr/share/openqa/script/client and /usr/bin/openqa-client

Reply:
I was referring to the point of `-cli` vs. `-client` above, so I'm not sure how `../script/client` relates here.
I also don't mind going with `openqa-cli` personally.

Reply:
@kalikiana sorry, I think @okurz was referring to my comment that I made because I was confused, and then I deleted it

Reply:
Yeah, that's confusing :)